package.json

```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/processing/p5.js-sound.git"
   },
-  "version": "0.3.12",
+  "version": "1.0.0",
```

Comment:
I'm not sure if this feels like a 1.0.0 release, but maybe it's time! Especially if we are going to follow the rules of semver with this non-backwards compatible change (though we haven't been great at following semver for previous releases!).

Reply:
Waving the green flag then ✨.
Off we go 😄❤️